Task_10/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style1.css" type="text/css"/>
+</head>
+<body>
+    <header>
+
+    <div class="conteiner_heder">
+        <img src="./img/rabbit-mouth.png" class="header_rabbit" >
+     <div class="header_text">
+         <span class="orange">orange</span><br>
+         <span class="rabbit_text4" >RABBIT</span>
```

Comment:
What is rabbit_text4?) Please use understandable class names.
Or you can use pseudo classes

Reply:
not fixed